src/components/look-controls.js

```diff
@@ -88,6 +91,12 @@ module.exports.Component = registerComponent('look-controls', {
     canvasEl.addEventListener('touchstart', this.onTouchStart);
     canvasEl.addEventListener('touchmove', this.onTouchMove);
     canvasEl.addEventListener('touchend', this.onTouchEnd);
+
+    // Pointer lock
+    document.addEventListener('pointerlockchange', this.onPointerLockChange, false);
+    document.addEventListener('pointerlockerror', this.onPointerLockError, false);
+    document.addEventListener('mozpointerlockchange', this.onPointerLockChange, false);
```

Comment:
FYI, I just checkend and Firefox seems to fire both `mozpointerlockchange` and `pointerlockchange` (ugh).